error_reporting/unit_tests/test_client.py

```diff
@@ -128,6 +128,32 @@ def test_report(self):
         self.assertLess(report_location['lineNumber'], 150)
 
 
+class TestFlaskHelper(unittest.TestCase):
+
+    def test_flask_helper(self):
+        from google.cloud.error_reporting import build_flask_context
+        request = _Request('http://google.com', 'GET',
+                           'Google Cloud Unit Tests Agent', 'http://gmail.com',
+                           '127.0.0.1')
+        context = build_flask_context(request)
+        self.assertEqual(request.url, context.url)
+        self.assertEqual(request.method, context.method)
+        self.assertEqual(request.user_agent, context.userAgent)
+        self.assertEqual(request.referrer, context.referrer)
+        self.assertEqual(request.remote_addr, context.remoteIp)
+
+
+# Fake request so no need to bring in Werkzeug as dependency
```

Comment:
The comment is not needed, it's obvious that this class is a mock. If you wanted it to be a little bit more obvious you could name it `_WerkzeugRequest`